src/dune_rules/module.mli

```diff
@@ -88,6 +88,7 @@ module Obj_map : sig
 end
 
 val sources : t -> Path.t list
+val orig_sources : t -> Path.t list
```

Comment:
`original_sources`. Or better yet, `sources_without_pp`. The originality of a source isn't descriptive without context